src/python/test/sugar/test_context.py

```diff
@@ -67,3 +69,6 @@ def test_cd_targetpath(self):
             self.assertNotEqual(prev_dir, curr_dir, "Failed to change dirs")
 
         self.assertTrue(os.path.exists("/tmp/some_path"), "Temp path should still exist")
```

Comment:
Must have missed this in a prior commit, but this test will fail to execute on non-unix systems. I recommend using the more portable [`tempfile.gettempdir`](https://docs.python.org/2/library/tempfile.html#tempfile.gettempdir). Something like:


```python
prev_dir = os.getcwd()
temp_dir = os.path.join(tempfile.gettempdir(), "some_path")

with context.cd(temp_dir, cleanup=False):
            curr_dir = os.getcwd()
            self.assertNotEqual(prev_dir, curr_dir, "Failed to change dirs")

        self.assertTrue(os.path.exists(temp_dir), "Temp path should still exist")

os.remove(temp_dir)
```